neo/IO/BinaryReader.py

```diff
@@ -139,7 +139,7 @@ def Read2000256List(self):
         items = []
         for i in range(0, 2000):
             data = self.ReadBytes(64)
-            ba = bytearray(binascii.unhexlify(data))
```

Comment:
The unhexlify is actually needed in this function. The data read from the stream, in the particular case that this function is used in, is non-raw data (as indicated by reading 64 bytes, as opposed to reading 32 bytes as in `ReadHashes()` below) i.e. `b'ef1f8f66a16fba100ed760f4ac6aa5a0d0bb8f4a0e92705b106761ef181718b3'`

When calling the `reverse()` function on this you'll get:
`b'3b817181fe167601b50729e0a4f8bb0d0a5aa6ca4f067de001abf61a66f8f1fe'`

while what actually is needed is:
`b'b3181718ef6167105b70920e4a8fbbd0a0a56aacf460d70e10ba6fa1668f1fef'`

Note that in the latter we reverse whole bytes, not just a single ascii char at a time.

Please revert the changes.

Reply:
Thanks for the feedback @ixje. I'm just now moving this to `neocore` and will also take a look later tonight.